--- /dev/null	2024-03-21 19:54:41.864001598 +0800
+++ BUILD	2024-03-23 10:08:47.086763650 +0800
@@ -0,0 +1,29 @@
+load("@rules_foreign_cc//foreign_cc:defs.bzl", "configure_make")
+
+package(default_visibility = ["//visibility:public"])
+
+filegroup(
+    name = "all_srcs",
+    srcs = glob(include = ["**"]),
+)
+
+configure_make(
+    name = "libmodsecurity",
+    configure_in_place = True,
+    configure_options = [
+        "--disable-doxygen-doc",
+        "--disable-doxygen-html",
+        "--disable-valgrind-memcheck",
+        "--disable-valgrind-helgrind",
+        "--disable-valgrind-drd",
+        "--enable-shared",
+        "--enable-static",
+    ],
+    args = [
+        "--jobs=$(nproc)",
+        "--load-average=$(nproc)",
+    ],
+    out_shared_libs = ["libmodsecurity.so"],
+    out_static_libs = ["libmodsecurity.a"],
+    lib_source = ":all_srcs",
+)
