--- /dev/null	2024-03-21 19:54:41.864001598 +0800
+++ BUILD	2024-03-23 10:07:58.338907824 +0800
@@ -0,0 +1,27 @@
+load("@rules_foreign_cc//foreign_cc:defs.bzl", "configure_make")
+
+package(default_visibility = ["//visibility:public"])
+
+filegroup(
+    name = "all_srcs",
+    srcs = glob(
+        include = ["**"],
+        exclude = ["*.bzl"],
+    ),
+)
+
+configure_make(
+    name = "libsodium",
+    configure_options = [
+        "--enable-shared=yes",
+        "--enable-static=yes",
+        "--with-pic",
+    ],
+    args = [
+        "--jobs=$(nproc)",
+        "--load-average=$(nproc)",
+    ],
+    lib_source = ":all_srcs",
+    out_shared_libs = ["libsodium.so"],
+    out_static_libs = ["libsodium.a"],
+)
